Source/configuration.txt

```diff
@@ -0,0 +1,21 @@
+configuration = {
```

Comment:
You'll want to consolidate this with the data files copy when you set everything up to build neatly.